library/Freckle/App/Kafka/Consumer.hs

```diff
@@ -167,7 +167,9 @@ runConsumer
   -> (a -> m ())
   -> m ()
 runConsumer pollTimeout onMessage =
-  withTraceIdContext $ immortalCreate onFinish $ do
+  withTraceIdContext $ immortalCreate onFinish messageLoop
```

Comment:
Why not just?

```suggestion
  withTraceIdContext $ immortalCreate onFinish $ forever $ do
```

Reply:
I always forget about `forever` for some reason...